lane.py

```diff
@@ -1049,6 +1049,8 @@ def search(self, _db, query, search_client, media=None, pagination=None, languag
 
         if not media:
             media = self.media
+        elif media is Edition.ALL_MEDIUM:
+            media = Edition.medium_to_additional_type.keys()
```

Comment:
This will work but it would be simpler (and possibly more efficient) to set `media=None` here. `ExternalSearchIndex.make_filter` will do the right thing.

Reply:
Updated. I tried that before but didn't work. I may have tried `None` elsewhere instead.